### ISSUE TITLE
Apparently normal trace writer initialization logs emitted as "warning" instead of "info"

### ISSUE THREAD

**Issue by julealgon** [open-telemetry/opentelemetry-collector-contrib#33053]
### Component(s)

exporter/datadog

## Description

"Trace writer initialized" logs that are emitted right after the collector is initialized are using the "warning" category. This doesn't seem to be a warning but a normal note and should use "information" instead.

> datadogexporter@v0.100.0/zaplogger.go:49	Trace writer initialized (climit=100 qsize=1)	{"kind": "exporter", "data_type": "traces", "name": "datadog"}

## Steps to Reproduce

Initialize the container and observe logs.

## Expected Result

I expected this log to be "information" instead of "warning" since it doesn't denote anything abnormal.

## Actual Result

A "warning" log is emitted for what appears to be a normal situation.

### Collector version

v0.100.0

## Environment
OS: Windows Server 2019 Datacenter v1809
Compiler(if manually compiled): N/A

### OpenTelemetry Collector configuration

```yaml
receivers:
  otlp:
    protocols:
      grpc:

  hostmetrics:
    collection_interval: 10s
    scrapers:
      paging:
        metrics:
          system.paging.utilization:
            enabled: true
      cpu:
        metrics:
          system.cpu.utilization:
            enabled: true
      disk:
      filesystem:
        metrics:
          system.filesystem.utilization:
            enabled: true
      load:
      memory:
      network:
      processes:

processors:
  batch:
    timeout: 10s

  resource:
    attributes:
    - key: deployment.environment
      value: "beta"
      action: upsert
  
exporters:
  datadog:
    api:
      key: ${DATADOG_APIKEY}

service:
  pipelines:
    traces:
      receivers: [otlp]
      processors: [resource, batch]
      exporters: [datadog]
    metrics:
      receivers: [hostmetrics, otlp]
      processors: [resource, batch]
      exporters: [datadog]
    logs:
      receivers: [otlp]
      processors: [resource, batch]
      exporters: [datadog]
```

### Log output

```shell
datadogexporter@v0.100.0/zaplogger.go:49	Trace writer initialized (climit=100 qsize=1)	{"kind": "exporter", "data_type": "traces", "name": "datadog"}
```

### Additional context

Found this by accident right after updating to v0.100.0. Not sure when it was actually introduced.

**Comment by mx-psi** [comment]
This is a bug upstream. DataDog/datadog-agent/pull/25625 fixes this, but it may take some time for the change to trickle down to the Datadog exporter

**Comment by mx-psi** [comment]
The PR has been merged upstream, so I am closing this :) This will be available in Agent 7.55, not before June, and in the Datadog exporter some time after that
